nntrainer/layers/common_properties.cpp

```diff
@@ -197,15 +197,15 @@ Padding2D::compute(const TensorDim &input, const TensorDim &kernel,
 
 bool Padding1D::isValid(const std::string &v) const {
 
-  /// case 1, 2: padding has string literal
-  if (istrequal(v, "valid") || istrequal(v, "same")) {
+  /// case 1, 2, 3: padding has string literal
+  if (istrequal(v, "valid") || istrequal(v, "same") || istrequal(v, "causal")) {
```

Comment:
I wonder if the casual option only works in 1D.

Reply:
According to the tensorflow con2d documentation, one of "valid" or "same" is available whreas "valid", "same" or "causal" is available in conv1d.
Please refer padding section in conv1d, conv2d
(conv1d: https://www.tensorflow.org/api_docs/python/tf/keras/layers/Conv1D, conv2d: https://www.tensorflow.org/api_docs/python/tf/keras/layers/Conv2D)

FYI in pytorch either a string {"valid", "same"} or tuple of int value is available in conv1d ("causal" is not available even in conv1d).

Reply:
right. thanks.